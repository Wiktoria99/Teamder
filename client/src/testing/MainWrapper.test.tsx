import { shallow} from "enzyme";
import {
  MainWrapper,
} from '@/components';
import toJson from "enzyme-to-json";
import { cleanup } from '@testing-library/react';

afterEach(cleanup);


it('EditInfo renders without crashing', () => {
  shallow(<MainWrapper />);
});

it("renders correctly", () => {
  const tree = shallow(<MainWrapper />);
  expect(toJson(tree)).toMatchSnapshot();
});
