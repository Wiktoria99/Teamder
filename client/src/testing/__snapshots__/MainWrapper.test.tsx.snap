// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <Styled(MuiBox)
    className="makeStyles-topPanel-1"
  >
    <h3
      className="makeStyles-welcome-4"
    >
      Witaj!
    </h3>
    <h3
      className="makeStyles-title-2"
    />
  </Styled(MuiBox)>
</Fragment>
`;
